cocotb/binary.py

```diff
@@ -284,11 +320,11 @@ def _adjust_twos_comp(self, x):
 
     def get_value(self):
         """Return the integer representation of the underlying vector."""
-        return self._convert_from[self.binaryRepresentation](self._str)
```

Comment:
This change means that users can no longer change `.binaryRepresentation` after the object is created. I don't know whether that was ever supported.

Reply:
It was. Changing `binaryRepresentation` after creation would result in a different result for `get_value` and all the `conversion` functions from that point forward. That being said, it was probably never a good idea, and it isn't documented.